Packs/Binalyze/README.md

```diff
@@ -2,7 +2,7 @@
 
 AIR is an "Automated Incident Response" platform that provides the complete feature set for:
 
-- Remotely collecting 150+ evidence types in minutes,
+- Remotely collecting 300+ evidence types in minutes,
```

Comment:
Is this change reflected in the API or should it be in the code?

Reply:
This change is unrelated to the API; since the creation of integration, our product's evidence capability has been improved. So this is why I changed, so it is just wording.